src/components/Agreements/ViewAgreement/Sections/Organizations.js

```diff
@@ -24,32 +25,66 @@ export default class Organizations extends React.Component {
     }).isRequired,
     id: PropTypes.string,
     onToggle: PropTypes.func,
+    organizations: PropTypes.object,
     open: PropTypes.bool,
   };
 
-  renderOrgList = (orgs) => {
+  renderOrgList = () => {
+    const { organizations } = this.props;
     return (
       <React.Fragment>
-        {orgs.map(o => (
-          o.org ?
-            <Layout data-test-organizations-org className="marginTopHalf" key={`${o.org.id}-${o.role.value}`}>
-              {o.org.orgsUuid ?
-                <Link to={`/organizations/view/${o.org.orgsUuid}`}>{o.org.name}</Link> :
-                o.org.name
-              }
-              {o.role && `, ${o.role.label}`}
-            </Layout> : null
-        ))}
+        {organizations && organizations.map(o => {
+          const interfaces = get(o, ['org', 'interfaces'], []);
+          return (
+            o.org ?
+              <Card
+                data-test-organizations-org
+                key={`${o.org.id}-${o.role.value}`}
+                header={
+                  <span className="my-card-header">
+                    {
+                      o.org.orgsUuid ?
+                        <span>
+                          <Link to={`/organizations/view/${o.org.orgsUuid}`}>
+                            {o.org.name}
+                          </Link>
+                          {o.role && ` · ${o.role.label}`}
```

Comment:
Ugh, sorry for not catching this till now, but this conditional probably makes more sense on line 53. We _always_ want to show the role, whether or not there is an `orgsUuid`.